ui/src/app/views/invitation-key/component.tsx

```diff
@@ -68,7 +68,7 @@ export class InvitationKey extends React.Component<InvitationKeyProps, Invitatio
   }
 
   render() {
-    return <div style={{padding: '3rem 3rem 0 3rem'}}>
+    return <div data-test-id='invitationKey' style={{padding: '3rem 3rem 0 3rem'}}>
```

Comment:
This should probably match the 'invitation' value used in the tests (currently obscured by `exists` issue)